#!/bin/sh /usr/share/dpatch/dpatch-run
## 211-magic-update-mono.dpatch
##
## DP: Update Mono magic.

@DPATCH@

diff -Naurp file.orig/magic/Magdir/msdos file/magic/Magdir/msdos
--- file.orig/magic/Magdir/msdos	2008-06-13 15:23:32.000000000 +0200
+++ file/magic/Magdir/msdos	2008-06-13 15:28:28.000000000 +0200
@@ -65,6 +65,7 @@
 >>&0	leshort		0x290	PA-RISC
 >>&18	leshort&0x0100	>0	32-bit
 >>&18	leshort&0x1000	>0	system file
+>>&228	lelong		>0	\b, Mono/.Net assembly
 >>&0xf4 search/0x140 \x0\x40\x1\x0
 >>>(&0.l+(4)) string MSCF \b, WinHKI CAB self-extracting archive
 >30		string	Copyright\ 1989-1990\ PKWARE\ Inc.	Self-extracting PKZIP archive
@@ -94,7 +95,7 @@
 >>>>(0x3c.l+4)	leshort		0x290	PA-RISC
 >>>>(0x3c.l+22)	leshort&0x0100	>0	32-bit
 >>>>(0x3c.l+22)	leshort&0x1000	>0	system file
->>>>(0x3c.l+232)	lelong	>0	Mono/.Net assembly
+>>>>(0x3c.l+232)	lelong	>0	\b, Mono/.Net assembly
 
 >>>>(0x3c.l+0xf8)	string		UPX0 \b, UPX compressed
 >>>>(0x3c.l+0xf8)	search/0x140	PEC2 \b, PECompact2 compressed
